### ISSUE TITLE
Problem building Diary page

### ISSUE THREAD

**Issue by mediapathic** [vimwiki/vimwiki#343]
When I issue the command to rebuild the diary page with new files I've created, I get the following error:

``` 
Error detected while processing function vimwiki#diary#generate_diary_section[5]..<SNR>169_form
at_diary[11]..<SNR>169_get_month_name:
line    1:
E716: Key not present in Dictionary: 41
E15: Invalid expression: g:vimwiki_diary_months[str2nr(a:month)]
E716: Key not present in Dictionary: 20
E15: Invalid expression: g:vimwiki_diary_months[str2nr(a:month)]   
```
I'm reasonably certain this is due to something I've done (perhaps an incompatibility with [taskwiki](https://github.com/tbabej/taskwiki) which I've been wrangling with?) and not an actual bug. But I'm posting here to get any hints as to where to start in debugging, as I'm lost. 

Thanks!

**Comment by mediapathic** [comment]
I've determined that this is not an incompatibility with taskwiki, or any other plugin (at least those installed by pathogen, which means anything I've installed in the last two years). I also created a clean .vimrc that only includes enough to run vimwiki, and I still get this error message. So this tells me, I think, that some file is confusing the generate diary function, but I have nothing but .txt files in the diary directory. 

Any ideas, anyone?
